clients/gitlabrepo/commits.go

```diff
@@ -92,13 +68,6 @@ func (handler *commitsHandler) setup() error {
 						mergeRequest = mergeRequests[i]
```

Comment:
This line will seg fault. `mergeRequest` is guaranteed to be `nil` when it reaches this line for the first time.

Reply:
I believe it's set on L62: `mergeRequest = mergeRequests[0]`. Or do you mean that's always nil?

Reply:
The first time this line is reached when all `mergeRequests[0:i-1]` are `nil`. So `mergeRequests[0] == mergeRequest == nil`.